### PR TITLE
feat(@clayui/empty-state): LPD-21741 Remove changes on Empty State and add SVG files

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

File: packages/clay-empty-state/src/index.tsx [hunk 1]
```diff
@@ -103,60 +85,47 @@ const ClayEmptyState = ({
...
 			})}
 			{...otherProps}
 		>
 			{hasImg && (
 				<div className="c-empty-state-image">
-					{imgSrc && (
-						<div className="c-empty-state-aspect-ratio">
+					<div className="c-empty-state-aspect-ratio">
+						<img
+							alt=""
```
Comment (matuzalemsteles) [inline packages/clay-empty-state/src/index.tsx:105]: Can you add it as part of the API to customize the alt?

File: packages/clay-empty-state/src/index.tsx [hunk 2]
```diff
@@ -6,35 +6,35 @@
...
 import React, {useMemo, useState} from 'react';
 
-import {States} from './States';
-
 interface IProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'title'> {
+	/**
+	 * Provides alternative information for an image if a user for some reason cannot view it
+	 * The default value is 'No results found'
+	 */
+	altProps?: string;
```
Comment (matuzalemsteles) [inline packages/clay-empty-state/src/index.tsx:14]: I think we can use the value of `imgProps`.

Comment (ilzamcmed) [reply packages/clay-empty-state/src/index.tsx]: We decided to remove the altProps and continue with `{...imgProps}`

**Review by matuzalemsteles — APPROVED** [review]
LGTM

**Pull request merged**